packages/core/src/accordion/accordion-header.element.ts

```diff
@@ -44,6 +44,10 @@ export class CdsAccordionHeader extends LitElement {
   connectedCallback() {
     super.connectedCallback();
     this.setAttribute('slot', 'accordion-header');
+
+    if (!this.id) {
```

Comment:
Same here. Should id be a lit property? Disregard if it already is. I'm reviewing on my phone and what I can see of the file is limited